packages/components/src/components/ScalarButton/ScalarButton.vue

```diff
@@ -15,11 +15,13 @@ withDefaults(
     loading?: LoadingState
     size?: Variants['size']
     variant?: Variants['variant']
+    type?: 'button' | 'submit'
```

Comment:
We should probably have the "reset" option as well since it's a valid option

Reply:
good call! added